wire-ios/Wire-iOS/Sources/UserInterface/Conversation Options/CreateSecureGuestLink/CreateSecureGuestLinkViewController.swift

```diff
@@ -122,14 +124,27 @@ class CreateSecureGuestLinkViewController: UIViewController, CreatePasswordSecur
         return textField
     }()
 
-    private var validationErrorTextColor = SemanticColors.Label.textErrorDefault
+    private lazy var createSecuredLinkButton: Button = {
+        let button = Button(
+            style: .primaryTextButtonStyle,
+            cornerRadius: 16,
+            fontSpec: .buttonBigSemibold
+        )
+        button.setTitle(SecuredGuestLinkWithPasswordLocale.CreateLinkButton.title, for: .normal)
+        button.addTarget(self, action: #selector(createSecuredLinkButtonTapped), for: .touchUpInside)
+        button.titleLabel?.numberOfLines = 0
+        return button
+    }()
+
+    private var createSecureGuestLinkPasswordValidatorHelper = CreateSecureGuestLinkPasswordValidatorHelper()
 
     // MARK: - Override methods
 
     override func viewDidLoad() {
         super.viewDidLoad()
         setUpViews()
         setupConstraints()
+        textFieldDidChange(securedGuestLinkPasswordTextfield)
```

Comment:
why do we call this in viewDidLoad?

Maybe better to have a private method making the configuration and `textFieldDidChange` calling this method

Reply:
The reason is for setting the state for the Create link button.